src/game.py

```diff
@@ -26,7 +28,7 @@ def __init__(self):
         self.outbreak_count = 0
         self.game_over = False
         self.game_won = False
-        self.city_map = {}
+        self.city_map = OrderedDict()
```

Comment:
I think it isn't necessary to use OrderedDict here as we haven't used city order anywhere in the code till now.
But it's ok.